lib/players/human.rb

```diff
@@ -0,0 +1,8 @@
+class Human < Player
+
+  def move(board=nil)
+    puts "Please select a position between 1-9: "
+    position = gets.chomp
```

Comment:
You don't need to store your return value in a local variable; just utilize the implicit return value of the expression.
```
def move(board=nil)
  puts "Please select a position between 1-9: "
  gets.chomp
end
```